Runtime/GameLiftCoreApi.cs

```diff
@@ -24,20 +25,21 @@ public class GameLiftCoreApi
     {
         private readonly GameLiftConfiguration _configuration;
         private readonly bool _isAnywhereMode;
-        private CoreApi _coreApi;
         private readonly IGameServerAdapter _gameServerAdapter;
+        private readonly CoreApi _coreApi;
 
         protected GameLiftCoreApi(GameLiftConfiguration configuration)
         {
             _configuration = configuration;
-            _coreApi = CoreApi.SharedInstance;
-            var fleetId = _coreApi.GetSetting("FleetId").Value;
-            var fleetLocation = _coreApi.GetSetting("FleetLocation").Value;
+            _coreApi = new CoreApi(Paths.PluginConfigurationFile);
```

Comment:
We still need to replace usage of `CoreApi` (and any other Editor code) in all runtime files

Reply:
replaced + removed dependency on editor assembly

Reply:
Have you verified if any other runtime scripts are using editor code? I only see you've changed this one file

Reply:
I have, and I also removed the dependency on Editor assembly from the Runtime assembly definitions